### ISSUE TITLE
Log games as "aborted" under ladder time upload threshold 

### ISSUE THREAD

**Issue by GrantBartlett** [CnCNet/leaderboard#16]
Instead of simply ignoring games below the ladder threshold, should we log them as "abandoned" or "aborted", possibly penalising the player who has left with - points, but not adding additional points to the opponent.

**Comment by sean3z** [comment]
IMO removing the threshold makes it (albeit slightly) easier for players to boost one another via free wins. 

@hifi @FunkyFr3sh Thoughts?

**Comment by GrantBartlett** [comment]
@sean3z - wouldn't want to remove it, just saying its possibly better to catch and log these early quits in the UI. 
- Person who quits - penalise for quitting. 
- Opponent gains no points, and is not penalised.

**Comment by FunkyFr3sh** [comment]
Good idea to show these games too.
Not sure what happens if one player doesn't connect cause of his firewall/anti-virus, before we think about removing points we would have to test such cases

**Comment by sean3z** [comment]
@FunkyFr3sh Is the client able to detect firewall settings prior to attempting to play (especially ladder) games? Perhaps prompting the user "Your settings are not optimal, please fix them before playing tourney games"?

@GrantBartlett I like that approach. Should the "person who quit" be considered as a disconnect? What kind of penalty (regular Elo [as if they lost] or a non-point penalty [increase disconnect count])?

**Comment by FunkyFr3sh** [comment]
Nope, cannot really know what the firewall is going to do after the game was started by the client
